l2geth/accounts/abi/abi.go

```diff
@@ -136,13 +138,17 @@ func (abi *ABI) UnmarshalJSON(data []byte) error {
 				_, ok = abi.Methods[name]
 			}
 			isConst := field.Constant || field.StateMutability == "pure" || field.StateMutability == "view"
-			abi.Methods[name] = Method{
+			method := Method{
```

Comment:
```suggestion
			method := Method {
```
Is this not linted?

Reply:
Having a space there is not the go-style, see:

https://github.com/ethereum-optimism/optimism/blob/35b99b07cf9f3e02496783c565b3f82f7fb55fef/l2geth/core/blockchain.go#L355